apps/server/cypress/e2e/suite1/testExample.cy.ts

```diff
@@ -38,6 +40,54 @@ const makeABConnection = async (jobType) => {
   expectConnectionSuccess();
 };
 
+const getAccountId = ({ aggregator, memberGuid, userId }) => {
+  const url = `/data/aggregator/${aggregator}/user/${userId}/connection/${memberGuid}/accounts`;
+
+  return cy.request("get", `/api${url}`).then((dataResponse) => {
+    const accountId = dataResponse.body.accounts.find(
+      (acc) => Object.keys(acc)[0] === "depositAccount",
+    ).depositAccount.accountId;
+
+    return accountId;
+  });
+};
+
+const verifyTransactionsValidatorSuccess = ({
+  accountId,
+  aggregator,
+  userId,
+}) => {
+  const url = `/data/aggregator/${aggregator}/user/${userId}/account/${accountId}/transactions${aggregator === TEST_EXAMPLE_B_AGGREGATOR_STRING ? "?start_time=2021/1/1" : ""}`;
```

Comment:
no reason for the aggregator === TEST_EXAMPLE_B_AGGREGATOR_STRING right?

Reply:
Yep. Fixed.